Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/ScheduleDatabase.java

```diff
@@ -155,11 +156,11 @@ public static ScheduleDetail fetchDetailData(@NonNull Context ctx, @NonNull Stri
                 item.talkTitle = c.getString(1);
                 item.photo = c.getString(2);
                 if (!c.isNull(3)) {
-                    item.time = c.getString(3);
+                    item.startTime = c.getString(3);
                     item.room = c.getString(4);
                     item.date = c.getString(5);
                 } else {
-                    item.time = "unscheduled";
+                    item.startTime = "unscheduled";
```

Comment:
"unscheduled" should be resource string

Reply:
I think we're axing this...it's the old sqlite stuff